src/types/index.ts

```diff
@@ -0,0 +1,22 @@
+export interface WatchtowerOptions {
+  contract: string;
+  publish: boolean;
```

Comment:
what is the publish option? maybe good to add some JS docs

Reply:
Tracking in #60